src/server/models/users.model.js

```diff
@@ -2,13 +2,12 @@
 
 import mongoose from 'mongoose';
 
-/**
+/*
  * Define the schema model for user.
  */
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
-  userId: String,
```

Comment:
@ethkim If there is no other option, the object that saved in collection gets `_id` from MongoDB. It's unique and can be found faster than custom numeric id (like AUTO_INCREMENT index in MySQL).

https://docs.mongodb.com/manual/reference/method/ObjectId/